File: app/src/main/java/com/digicoffer/lauditor/DocEditor/Structure_Payload/FieldContentModel.java
package com.digicoffer.lauditor.DocEditor.Structure_Payload;

import java.util.ArrayList;
import java.util.List;

public class FieldContentModel {
    public String fieldname;
    public String title;
    public String contentText;
    public String fileName;
    public List<ItemListModel> itemList = new ArrayList<>();
    public List<List<ItemListModel>> tableItemList = new ArrayList<>();
}

File: app/src/main/java/com/digicoffer/lauditor/DocEditor/Structure_Payload/FieldContentModel.kt
package com.digicoffer.lauditor.DocEditor.Structure_Payload

import java.util.ArrayList

class FieldContentModel {
    @JvmField
    var fieldname: String? = null
    
    @JvmField
    var title: String? = null
    
    @JvmField
    var contentText: String? = null
    
    @JvmField
    var fileName: String? = null
    
    @JvmField
    var itemList: MutableList<ItemListModel>? = ArrayList()
    
    @JvmField
    var tableItemList: MutableList<MutableList<ItemListModel>>? = ArrayList()
}
